bot.lua

```diff
@@ -4706,7 +4715,8 @@ commands["lua"] = {
 				getOwner(message, "getAllMembers")
 
 				local names, index = { }, 0
-				message.guild.members:findAll(function(member)
+				local guild = message.guild or client:getGuild(channels["guild"])
```

Comment:
Instead of duplicating the code why don't you make this variable outside the functions?